portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -37,60 +39,92 @@
+        createEntitys();
+    }
+    else if(request.getParameter(MAX_FORM) != null && !request.getParameter(MAX_FORM).isEmpty()) {
+      setMaxComment(request, request.getParameter(MAXCOMMENT_PARAMETER));
+    }
 
-    createEntitys();
     response.sendRedirect(COMMENT_HTML_PARAMETER);
   }
 
+  public void createEntitys(){
+    Entity commentEntity = new Entity(COMMENT_PARAMETER);
+    commentEntity.setProperty(USERNAME_PARAMETER, userNameInput);
+    commentEntity.setProperty(REFLECTION_PARAMETER, reflectionInput);
+    commentEntity.setProperty(TIMESTAMP_PARAMETER, timestampOfComment);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(commentEntity);
+  }
+
+  public void setMaxComment(HttpServletRequest request, String maxComment){
+    try {
+        this.maxComment = Integer.parseInt(maxComment);
+    } catch (NumberFormatException e) {
+        throw new NumberFormatException("Max comments cannot be converted to int: " + maxComment);
+    }
+  }
+
+  public int getMaxComment(){
+      return maxComment;
+  }
+
+  @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query(COMMENT_PARAMETER).addSort(TIMESTAMP_PARAMETER, SortDirection.DESCENDING);
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
-    populateList(results);
-
-    Gson gson = new Gson();
+    displayMaxComments(results, getMaxComment());
+    String json = new Gson().toJson(comments);
     response.setContentType(APPLICATION_JSON_PARAMETER);
-    response.getWriter().println(gson.toJson(this.comments));
+    response.getWriter().println(json);
   }
-
-  public void createEntitys(){
-    Entity commentEntity = new Entity(this.COMMENT_PARAMETER);
-    commentEntity.setProperty(this.USERNAME_PARAMETER, this.userNameInput);
-    commentEntity.setProperty(this.REFLECTION_PARAMETER, this.reflectionInput);
-    commentEntity.setProperty(this.TIMESTAMP_PARAMETER, this.timestampOfComment);
-    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    datastore.put(commentEntity);
+  
+  public void displayMaxComments(PreparedQuery results, int maxComment){
+    comments = new ArrayList<>();
+    for(Entity commentEntity : results.asIterable()){
+      if(maxComment == 0){break;}
+      comments.add(createCommentFromEntity(commentEntity));
+      maxComment--;
+    }
   }
 
-  public void populateList(PreparedQuery results){
-    this.comments = new ArrayList<>();
-    for (Entity commentEntity : results.asIterable()) {
-      Comment comment = new Comment(
+  public String getCurrentTime(){  
+   DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss");
+   LocalDateTime now = LocalDateTime.now();
+   return(dtf.format(now).toString());
+  }  
```

Comment:
This can be used to just create the comments, and can be renamed to `getCurrentTimeString`